gcc-checker-plugin.spec

```diff
@@ -0,0 +1,23 @@
+### RPM external gcc-checker-plugin 1.0
```

Comment:
@gartung , use version 2.0.9 here and replace 2.0.9 with %{realversion} in rest of the file.